### ISSUE TITLE
embed URL has extra parameters

### ISSUE THREAD

**Issue by ChipDale66** [MicrosoftDocs/powerbi-docs#2710]
the secure embed code is obtained from a new menu item called Share, it is no longer on the ... menu

Also, the URL has extra parameters to those shown in your example

the screen shot of your example has the URL showing just a reportId and &autoAuth=true
e.g. https://app.powerbi.com/reportEmbed?reportId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx&autoAuth=true

however, when I get the embed code it has extra parameters, &ctid and &config and &appId for a report in an APP, or &groupId for a report from a workspace

https://app.powerbi.com/reportEmbed
?reportId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&appId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&autoAuth=true
&ctid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&config=12345678901234567890123456789012345678901234

https://app.powerbi.com/reportEmbed
?reportId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&groupId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&autoAuth=true
&ctid=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
&config=12345678901234567890123456789012345678901234

I published a link in December and it now does not work as the &config value is different.

What do these parameters do?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7197c1f9-ff31-f86c-faf3-ab89e4ab9573
* Version Independent ID: f30cdf40-43c4-e0c3-4941-a02f5bb28fc1
* Content: [Embed a report in a secure portal or website - Power BI](https://docs.microsoft.com/en-us/power-bi/collaborate-share/service-embed-secure)
* Content Source: [powerbi-docs/collaborate-share/service-embed-secure.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/collaborate-share/service-embed-secure.md)
* Service: **powerbi**
* Sub-service: **pbi-collaborate-share**
* GitHub Login: @maggiesMSFT
* Microsoft Alias: **maggies**

**Comment by maggiesMSFT** [comment]
Hi @ChipDale66 -- Thanks for pointing that out! I'll update the screen shot, and get answers about the additional parameters.

**Comment by maggiesMSFT** [comment]
Hi, @ChipDale66 -- here's what I found out:
Old secure embed URLs should keep working regardless of those parameters. 
The config parameter is an internal parameter. It should not affect the functionality of the report load, but makes it slightly faster. 
You may need to file a support ticket, https://powerbi.microsoft.com/en-us/support/
Thanks

**Comment by ChipDale66** [comment]
Hi @maggiesMSFT

I did log a support call - 121010425005408 - and was told this is an unfortunate consequence of an improvement to load balancing. This is very unhelpful so I have had to go and update my links. 

The &config parameter points at a cluster URL. However it seems this parameter is optional so if I leave it off the URL links still works. Is this correct and what are the implications? If this parameter changes again because the system has updated the cluster URL then the links would break again. I think this needs to be covered on this page. Are you also going to explain the &ctid, which apparently was introduced to cover cross-tenancy reports, and the &appid and &groupid as well?

Many thanks
Andrew
